dashboard/modules/node/tests/test_node.py

```diff
@@ -70,6 +70,7 @@ def test_node_info(disable_aiohttp_cache, ray_start_with_dashboard):
     @ray.remote
     class Actor:
         def getpid(self):
+            print(f"actor pid={os.getpid()}")
```

Comment:
remove?

Reply:
I wanted to keep this message, or is it too noisy?